tests/tests_selinux_modules.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: all
+  become: true
+  become_method: sudo
+  become_user: root
+  vars:
+    selinux_modules:
+      - { path: "linux-system-roles-selinux-test-a.cil" }
+      - { name: "linux-system-roles-selinux-test-a", state: "disabled" }
+      - { name: "linux-system-roles-selinux-test-a", state: "enabled" }
+      - { path: "linux-system-roles-selinux-test-b.cil", priority: "500" }
+      - { name: "linux-system-roles-selinux-test-b", priority: "500", state: "disabled" }
+      - { path: "linux-system-roles-selinux-test-c.cil", priority: "600" }
+      - { name: "linux-system-roles-selinux-test-c", priority: "600", state: "absent" }
+      - { name: "linux-system-roles-selinux-test-d", state: "disabled" }
+
+  tasks:
+    - name: execute the role and catch errors
+      block:
+        - include_role:
+            name: linux-system-roles.selinux
```

Comment:
are you missing a `rescue` here?

Reply:
https://github.com/linux-system-roles/selinux/pull/56/commits/30754030aef1ba1e37b8dd8c1688d59b768e672d